std/experimental/ndslice/slice.d

```diff
@@ -638,6 +638,16 @@ auto makeSlice(T,
 }
 
 /// ditto
+auto makeSlice(
+    Flag!`replaceArrayWithPointer` replaceArrayWithPointer = Yes.replaceArrayWithPointer,
```

Comment:
So... where can I find docs for replaceArrayWithPointer? https://www.google.com/webhp?sourceid=chrome-instant&ion=1&espv=2&ie=UTF-8#q=replacearraywithpointer doesn't help.